projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/MultiProgress.svelte

```diff
@@ -64,6 +68,8 @@
 						isLink={hasIssues}
 						class="jb-navigator-link"
 						to="/image-size-analysis/{group}/1"
+						trackEvent="clicked_isa_settings_group"
+						trackEventProps="group"
```

Comment:
Using string `group` instead of group slug.

Reply:
Thanks, you're right!